webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -460,6 +478,10 @@ Kommunicate.richMsgEventHandler = {
     },
     handleFormSubmit: function(e) {
         e.preventDefault();
+    },
+    isValidString: function (string, value) {
```

Comment:
it is recommended to avoid using names of JavaScript built-in objects, properties, and methods
https://www.w3schools.com/js/js_reserved.asp

Reply:
@devashishmamgain variable renamed